plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy_test.go

```diff
@@ -192,19 +196,46 @@ func TestBootstrapClusterRoleBindings(t *testing.T) {
 }
 
 func TestBootstrapControllerRoles(t *testing.T) {
-	list := &api.List{}
-	names := sets.NewString()
-	roles := map[string]runtime.Object{}
-	bootstrapRoles := bootstrappolicy.ControllerRoles()
-	for i := range bootstrapRoles {
-		role := bootstrapRoles[i]
-		names.Insert(role.Name)
-		roles[role.Name] = &role
+	tests := map[string]struct {
+		filename         string
+		enabledFeatures  []featuregate.Feature
+		disabledFeatures []featuregate.Feature
+	}{
+		"PodDisruptionConditions enabled": {
+			enabledFeatures: []featuregate.Feature{features.PodDisruptionConditions},
+			filename:        "controller-roles-PodDisruptionConditions-enabled.yaml",
+		},
+		"PodDisruptionConditions disabled": {
+			disabledFeatures: []featuregate.Feature{features.PodDisruptionConditions},
+			filename:         "controller-roles-PodDisruptionConditions-disabled.yaml",
```

Comment:
you could just use `controller-roles.yaml` here. If we need a new file in the future, we'll deal with it at that point.

Reply:
indeed, done